error.go

```diff
@@ -38,10 +39,23 @@ func (e *errResponse) Response() *http.Response {
 	return e.response
 }
 
-// NewError ...
-func NewError(expectedRespCode []int, resp *http.Response) Error {
+// NewResponseError ...
+func NewResponseError(expectedRespCode []int, resp *http.Response) Error {
 	return &errResponse{
 		expectedResponseCode: expectedRespCode,
 		response:             resp,
 	}
 }
+
+// ScreeningErrors ...
+type ScreeningErrors struct {
```

Comment:
@chandradeepak you'll still need a way to iterate the errors since it's not exported. I recommend adding an `Errors() []error` fund to achieve this.

Reply:
@magicalbanana  made it 
type ScreeningErrors []error
so that we can iterate on it and then we don't even need extra Errors() function